src/renderer/account/components/SendPanel/index.js

```diff
@@ -31,6 +36,11 @@ export default compose(
 
   withData(authActions, mapAuthDataToProps),
   withNetworkData(),
+  withInitialCall(balancesActions, ({ net, address }) => ({ net, address })),
+  withData(balancesActions, mapBalancesDataToProps),
+  withProps(({ balances }) => ({
+    balances: pickBy(balances, ({ balance }) => balance !== '0')
```

Comment:
We're never using the keys from this object in the component.  I think we can just pass the values here instead (i.e.: just wrap this in a `values(...)` call).  That will also require changing the propType from `objectOf(...)` to `arrayOf(...)`.